tools/SampleAnalyzer/Interfaces/substructure/README.md

```diff
@@ -69,16 +84,19 @@ of jet definitions. All available `jet_algorithm` definitions are shown with the
 |      `cdfjetclu`      | `radius=0.4`, `ptmin=5.`, `overlap=0.5`, `seed=1.`, `iratch=0.`                    |
 |     `cdfmidpoint`     | `radius=0.4`, `ptmin=5.`, `overlap=0.5`, `seed=1.`, `iratch=0.`, `areafraction=1.` |
 |       `siscone`       | `radius=0.4`, `ptmin=5.`, `overlap=0.5`, `input_ptmin=5.`, `npassmax=1.`           |
-|      `VariableR`      | `rho=2000.`, `minR=0.`, `maxR=2.`, `ptmin=20.` `exclusive=False` `clustertype=CALIKE` `strategy=Best` |
+|      `VariableR`      | `rho=2000.`, `minR=0.`, `maxR=2.`, `ptmin=20.` `exclusive=False`                   |
+|                       | `clustertype=CALIKE` `strategy=Best`                                               |
```

Comment:
separating a table line like this creates a separate line, please let markdown take care of the line handling
```
|      `VariableR`      | `rho=2000.`, `minR=0.`, `maxR=2.`, `ptmin=20.` `exclusive=False`                   |
 |                       | `clustertype=CALIKE` `strategy=Best`                                               |
``` 
needs to be 
```
|      `VariableR`      | `rho=2000.`, `minR=0.`, `maxR=2.`, `ptmin=20.` `exclusive=False` `clustertype=CALIKE` `strategy=Best` |
```